scripts/download_image.py

```diff
@@ -187,6 +188,29 @@ def main(argv: list):
     parser = _create_parser()
     options = parser.parse_args(argv)
 
+    if options.site == 'game' and options.workers > GAME_WORKER_LIMIT:
+        if options.y:
+            logging.info("Passing warning message...")
+        else:
```

Comment:
- single name 'y' can be long name
  - choice y/n is ok there, users does not want input long.

this section can be another function? because now it's a long else: block

```python
if yes:
    logging.info("Passing warning message...")
    return
choid = input(
```

Reply:
Changed main method to this:
```
    if options.site == 'game' and options.workers > GAME_WORKER_LIMIT:
        if options.y:
            logging.info("Passing warning message...")
        elif show_game_download_warning(options.workers):
            return
```
And show_game_download_warning is:
```
def show_game_download_warning(workers: int,
                               input_fn: callable = input) -> bool:
    """
    Shows warning in case user selected game as a source
    :param input_fn: Input function for test or file reading
    :param workers: Number of workers
    :return:
    >>> show_game_download_warning(10, input_fn=lambda x: "y")
    False
    >>> show_game_download_warning(10, input_fn=lambda x: "n")
    True
    """
    choice = input_fn(
            'WARNING: You specified {} workers which is over limit {'
            '} to '
            'download images from game website. This may lead you to '
            'get '
            'banned. If you accept it enter Y or enter N to exit '
            'script. '
            'If not please define number of threads with -w '
            'parameter. '
            'If you want to skip this warning you can enter -y while '
            'calling script.:'.format(workers,
                                      GAME_WORKER_LIMIT)).lower()
    while choice not in ['y', 'n']:
        choice = input_fn('Either enter Y or N as an argument:').lower()
    if choice == 'n':
        return True
    else:
        logging.info('Warning accepted, script will continue')
        return False
```

Reply:
NO return True? how about caller change 
if not accept_game_download_warning(): ...

Reply:
Fixed